client/state/current-user/gravatar-status/selectors.js

```diff
@@ -11,3 +11,15 @@ import { get } from 'lodash';
 export function isCurrentUserUploadingGravatar( state ) {
 	return get( state, 'currentUser.gravatarStatus.isUploading', false );
 }
+
+/**
+ * Returns the temp Gravatar if it exists and
+ * the current user ID is passed in, otherwise false
+ * @param {Object} state - The state
+ * @param {Number} userId - The ID of the user we're checking
+ * @returns {String|Boolean} - The temp Gravatar string, or false
+ */
+export function getUserTempGravatar( state, userId ) {
```

Comment:
On second thought, I suppose we're optimizing for upcoming changes to the Gravatar piece to avoid an isCurrentUser check.

We can hold off on changes until we see an actual usage